src/components/Navbar/TrmWalletScreeningAlert.tsx

```diff
@@ -0,0 +1,50 @@
+import {
+  Alert,
+  AlertDescription,
+  AlertIcon,
+  CloseButton,
+} from "@chakra-ui/react"
+import { FC, useEffect, useState } from "react"
+import { RootState } from "../../store"
+import { useSelector } from "react-redux"
+
+const TrmWalletScreeningAlert: FC = () => {
+  const { isBlocked } = useSelector((state: RootState) => state.account.trm)
+  const [hideAlert, setHideAlert] = useState(false)
+  const [alertDescription, setAlertDescription] = useState("")
+
+  useEffect(() => {
+    if (!isBlocked) {
+      setHideAlert(true)
+      return
+    }
+
+    if (isBlocked) {
+      setAlertDescription(
+        `Your wallet has been flagged in our risk assessment screening. The 
+        Contract interactions are currently disabled.`
+      )
+      setHideAlert(false)
+      return
+    }
+  }, [isBlocked])
+
+  const resetAlert = () => {
+    setHideAlert(true)
+    setAlertDescription("")
+  }
+
+  if (hideAlert) {
+    return null
+  }
+
+  return (
+    <Alert status="error" variant="solid">
+      <AlertIcon />
```

Comment:
Same as above for this alert

```suggestion
    <Alert status="error" variant="solid" width="fit-content">
      <AlertIcon alignSelf="center" />
```

Reply:
Addressed in [apply styling to alert related components](https://github.com/threshold-network/token-dashboard/pull/736/commits/39e6e2d006412856ad90c9c80e4f9d712ea3d6e6)